libs/ui/src/remove_card_page.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Main } from './main';
+import { Prose } from './prose';
+import { fontSizeTheme } from './themes';
+
+const RemoveCardImage = styled.img`
+  margin-right: auto;
+  margin-bottom: 5px;
+  margin-left: auto;
+  height: 20vw;
+`;
+
+export function RemoveCardPage(): JSX.Element {
+  return (
+    <Main padded centerChild>
+      <RemoveCardImage src="remove-card.svg" alt="Remove Card Icon" />
+      <Prose textCenter theme={fontSizeTheme.medium} maxWidth={false}>
+        <h1>Successfully Authenticated.</h1>
+        <p>Remove Card.</p>
```

Comment:
To match the text styling conventions used elsewhere, e.g.

<img width="300" alt="styling" src="https://user-images.githubusercontent.com/12616928/174892196-0fd318b2-f5fb-4c8d-a00f-6d802105ab92.png">

let's remove the period after the "Successfully Authenticated" title and sentence case "Remove card."